ratelimits/limit.go

```diff
@@ -35,26 +39,32 @@ type LimitConfig struct {
 
 type LimitConfigs map[string]*LimitConfig
 
-// limit defines the configuration for a rate limit or a rate limit override.
+// Limit defines the configuration for a rate Limit or a rate Limit override.
 //
-// The zero value of this struct is invalid, because some of the fields must
-// be greater than zero.
-type limit struct {
-	// burst specifies maximum concurrent allowed requests at any given time. It
+// The zero value of this struct is invalid, because some of the fields must be
+// greater than zero. It and several of its fields are exported to support admin
+// tooling used during the migration from overrides.yaml to the overrides
+// database table.
+type Limit struct {
+	// Burst specifies maximum concurrent allowed requests at any given time. It
 	// must be greater than zero.
-	burst int64
+	Burst int64
 
-	// count is the number of requests allowed per period. It must be greater
+	// Count is the number of requests allowed per period. It must be greater
 	// than zero.
-	count int64
+	Count int64
 
-	// period is the duration of time in which the count (of requests) is
+	// Period is the duration of time in which the count (of requests) is
 	// allowed. It must be greater than zero.
-	period config.Duration
+	Period config.Duration
 
-	// name is the name of the limit. It must be one of the Name enums defined
+	// Name is the Name of the limit. It must be one of the Name enums defined
```

Comment:
The second instance of `Name` here should be lower-cased.

Reply:
Same here!